docs/customservices.md

```diff
@@ -272,4 +272,9 @@ Configuration example:
   warning_value: 50
   danger_value: 80
   api_token: "PVEAPIToken=root@pam!your-api-token-name=your-api-token-key"
+  # values below this line are optional (default value are false/empty):
+  hide_decimals: true # removes decimals from stats values.
+  hide: ["vms", "vms_total", "lxcs", "lxcs_total", "disk", "mem", "cpu"] # hides values included in the array
```

Comment:
I think it could be better to me to choose what to show, what do you think ?

Reply:
I think most users want to show as much info as possible, that's why I thought it was better to put what you want to hide. Following that line, you don't have to add an extra config line if you want to show all info.

Anyhow, I can change the implementation if your think it's better to use show instead of hide. In that case, I would change everything as I don't like to mix show and hide fields. Your choice :)